django/bossingest/views.py

```diff
@@ -227,16 +228,17 @@ def track_usage_data(self, ingest_config_data):
         Raises:
             (BossError): if user doesn't have permission for a large ingest.
         """
-
-        # ToDo: handle volumetric ingests.  Likely that first version of ingest
-        # schema doesn't have ingest type so check for tile-size to confirm 
-        # job is a tile ingest.
-        if 'tile-size' not in ingest_config_data['ingest_job']:
-            return
+        blog = bossLogger()
+        
+        # need to get bytes per pixel to caculate ingest in total bytes
+        ingest_job = ingest_config_data['ingest_job']
```

Comment:
Given that this comes from a config file created by a user, we probably want to wrap this in `try-except` in case the user enters bad data. In case GitHub doesn't highlight all the lines with this comment, I'm referring to lines 234-242.

Reply:
Good point. I think we can default to 8 bit words?